stats.rst

```diff
@@ -0,0 +1,307 @@
+  $ sudo chown -R _graphite:_graphite /var/log/graphite/
+
+^^^^^^^^^^^^^^^^^^
+Installing Apache2
+^^^^^^^^^^^^^^^^^^
+
+The next thing you need to run Graphite's Web app is a web server. Here I'll show you how to set up ``Apache2``.
+
+.. code-block:: console
+
+  $ sudo apt-get install -y apache2 libapache2-mod-wsgi-py3
+  $ sudo cp /usr/share/graphite-web/apache2-graphite.conf /etc/apache2/sites-available
+  $ sudo a2dissite 000-default
+  Site 000-default disabled.
+  $ sudo a2ensite apache2-graphite
+  Enabling site apache2-graphite.
+  $ sudo systemctl reload apache2
+
+If everything went well you should be able to see the graphite dashboard by going to ``127.0.0.1`` or ``0.0.0.0`` in your web browser
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Configuring Graphite (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default Graphite will plot points once per minute. For use within flux, one minute is a long time and often causes some loss of information.
+To change the interval at which Graphite (really carbon) is going to be plotting data there are a few things you need to do.
+
+First make sure that the carbon cache is enable on boot.
+
+.. code-block:: console
+
+  $ sudo vim /etc/default/graphite-carbon
+  # set to true
+  CARBON_CACHE_ENABLED=true
+
+Next, you will need to edit carbon's config file and set the specific you want. You can find specifics on the configurations `here <https://graphite.readthedocs.io/en/stable/config-carbon.html>`_.
+The carbon config file is stored in the ``CONF_DIR`` that was found earlier.
+
+.. code-block:: console
+
+  $ sudo vim /etc/carbon/storage-schemas.conf
+  # comment out this default
+  #[default_1min_for_1day]
+  #pattern = .*
+  #retentions = 60s:1d
+
+  # catch all metrics that start with 'flux'
+  # and retain each data point as 1s
+  [flux]
+  pattern = ^flux.*
+  retentions = 1s:3d
+
+Then you will need to clear out any existing whisper files in the database which is store in the ``LOCAL_DATA_DIR`` found earlier.
+
+.. code-block:: console
+
+  $ sudo rm -rf /val/lib/graphite/whisper/*
+
+Now restart the carbon daemon and the new configuration should take effect.
+
+.. code-block:: console
+
+  $ sudo systemctl restart carbon-cache
+
+--------------------------------
+Running Stats Collection in Flux
+--------------------------------
+
+Once you have brubeck and Graphite setup and Graphite running there are only two more things left to be able to get stats from ``flux``.
+First you will need to set the ``FLUX_FRIPP_STATSD`` environment variable with the endpoint of the aggregator (brubeck in this case).
+If you just used the default config for brubeck as suggested earlier, the endpoint is ``0.0.0.0:8126``. If you used the docker setup the
+endpoint is ``0.0.0.0:8125``.
+
+.. code-block:: console
+
+  $ export FLUX_FRIPP_STATSD=hostname:port
+
+^^^^^^^^^^^^
+Manual Setup
+^^^^^^^^^^^^
+
+The second thing needed is to run brubeck. There are a few option, but no real best way as of now.
```

Comment:
It should be mentioned here that, when using the graphiteapp docker image, `brubeck` is already running inside of that image. Perhaps this section should be named `Optional: launch brubeck inside of a Flux instance` or something.

I would say you could even replace this section with a simple "Note" that says, "if brubeck is not already running, for example in the graphite docker image or started outside of any Flux instance, then it will need to be started now".

We can perhaps leave the open question of how to start a per-instance brubeck to the reader (especially in the docker case, that will be a little tricky and perhaps beyond the scope of a quickstart guide)